src/main/java/org/broadinstitute/consent/http/service/VoteService.java

```diff
@@ -177,13 +178,15 @@ private boolean isDacChairPerson(Dac dac, DACUser user) {
         if (dac != null) {
             return user.getRoles().
                     stream().
-                    anyMatch(userRole ->
+                    anyMatch(userRole -> Objects.nonNull(userRole.getRoleId()) &&
+                            Objects.nonNull(userRole.getDacId()) &&
                             userRole.getRoleId().equals(UserRoles.CHAIRPERSON.getRoleId()) &&
-                                    userRole.getDacId().equals(dac.getDacId()));
+                            userRole.getDacId().equals(dac.getDacId()));
```

Comment:
This block was the trigger for the problem we saw in staging. In the failure case, the user had a chairperson role with no associated dac. That threw  an NPE and failed to create votes, but left the user in a state where they were a valid dac member, but didn't have a vote to display in the UI.